test/synth/synth-stdout.test.ts

```diff
@@ -1,7 +1,314 @@
-// eslint-disable-next-line
-const cmd = require('../../src/cli/cmds/synth');
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as yaml from 'yaml';
+import { Config, ValidationConfig } from '../../src/config';
+import { mkdtemp } from '../../src/util';
+
+beforeEach(() => {
```

Comment:
Is this to reset stdout?

Reply:
Nope. Its to reset `require.cache` so that every test can reacquire the `../../src/cli/cmds/synth` module, as if it was the first time, just like a CLI invocation.